SU2_CFD/src/fluid/CSU2TCLib.cpp

```diff
@@ -578,6 +565,436 @@ CSU2TCLib::CSU2TCLib(const CConfig* config, unsigned short val_nDim, bool viscou
+    ArrheniusTheta[10]  = 59500.0;
+    ArrheniusTheta[11]  = 59500.0;
+    ArrheniusTheta[12] = 75500.0;
+    ArrheniusTheta[13] = 75500.0;
+    ArrheniusTheta[14] = 75500.0;
+    ArrheniusTheta[15] = 75500.0;
+    ArrheniusTheta[16] = 75500.0;
+    ArrheniusTheta[17] = 75500.0;
+    ArrheniusTheta[18] = 38400.0;
+    ArrheniusTheta[19] = 19450.0;
+    ArrheniusTheta[20] = 31900.0;
+    ArrheniusTheta[21] = 113200.0;
+
+    /*--- Set rate-controlling temperature exponents ---*/
+    //  -----------  Tc = Ttr^a * Tve^b  -----------
+    //
+    // Forward Reactions
+    //   Dissociation:         a = 0.5, b = 0.5  (OR a = 0.7, b =0.3)
+    //   Exchange:             a = 1,   b = 0
+    //   Associative ion...    a = 1,   b = 0  ???
+    //   E Impact dissociation a = 0,   b = 1  
+    //   E Impact ionization:  a = 0,   b = 1
+    //
+    // Backward Reactions
+    //   Dissociation:           a = 1,   b = 0
+    //   Exchange:               a = 1,   b = 0
+    //   Associative  ion...     a = 0.5, b = 0.5
+    //   E Impact ionization:    a = 0,   b = 1
+    //   E Impact dissocitation: a = 0.5, b = 0.5 ???
+    //   N2 impact dissociation: a = 0,   b = 1
+    //   Others:                 a = 1,   b = 0
+    Tcf_a[0]  = 0.5; Tcf_b[0]  = 0.5; Tcb_a[0]  = 1;   Tcb_b[0] = 0;
+    Tcf_a[1]  = 0.5; Tcf_b[1]  = 0.5; Tcb_a[1]  = 1;   Tcb_b[1] = 0;
+    Tcf_a[2]  = 0.5; Tcf_b[2]  = 0.5; Tcb_a[2]  = 1;   Tcb_b[2] = 0;
+    Tcf_a[3]  = 0.5; Tcf_b[3]  = 0.5; Tcb_a[3]  = 1;   Tcb_b[3] = 0;
+    Tcf_a[4]  = 0.5; Tcf_b[4]  = 0.5; Tcb_a[4]  = 1;   Tcb_b[4] = 0;
+    Tcf_a[5]  = 0.5; Tcf_b[5]  = 0.5; Tcb_a[5]  = 1;   Tcb_b[5] = 0;
+    Tcf_a[6]  = 0.5; Tcf_b[6]  = 0.5; Tcb_a[6]  = 1;   Tcb_b[6] = 0;
+    Tcf_a[7]  = 0.5; Tcf_b[7]  = 0.5; Tcb_a[7]  = 1;   Tcb_b[7] = 0;
+    Tcf_a[8]  = 0.5; Tcf_b[8]  = 0.5; Tcb_a[8]  = 1;   Tcb_b[8] = 0;
+    Tcf_a[9]  = 0.5; Tcf_b[9]  = 0.5; Tcb_a[9]  = 1;   Tcb_b[9] = 0;
+    Tcf_a[10] = 0.5; Tcf_b[10] = 0.5; Tcb_a[10] = 1;   Tcb_b[10] = 0;
+    Tcf_a[11] = 0.5; Tcf_b[11] = 0.5; Tcb_a[11] = 1;   Tcb_b[11] = 0;
+    Tcf_a[12] = 0.5; Tcf_b[12] = 0.5; Tcb_a[12] = 1;   Tcb_b[12] = 0;
+    Tcf_a[13] = 0.5; Tcf_b[13] = 0.5; Tcb_a[13] = 1;   Tcb_b[13] = 0;
+    Tcf_a[14] = 0.5; Tcf_b[14] = 0.5; Tcb_a[14] = 1;   Tcb_b[14] = 0;
+    Tcf_a[15] = 0.5; Tcf_b[15] = 0.5; Tcb_a[15] = 1;   Tcb_b[15] = 0;
+    Tcf_a[16] = 0.5; Tcf_b[16] = 0.5; Tcb_a[16] = 1;   Tcb_b[16] = 0;
+    Tcf_a[17] = 0.5; Tcf_b[17] = 0.5; Tcb_a[17] = 1;   Tcb_b[17] = 0;
+    Tcf_a[18] = 1.0; Tcf_b[18] = 0.0; Tcb_a[18] = 1;   Tcb_b[18] = 0;
+    Tcf_a[19] = 1.0; Tcf_b[19] = 0.0; Tcb_a[19] = 1;   Tcb_b[19] = 0;
+    Tcf_a[20] = 1.0; Tcf_b[20] = 0.0; Tcb_a[20] = 0.5; Tcb_b[20] = 0.5;
+    Tcf_a[21] = 0.0; Tcf_b[21] = 1.0; Tcb_a[21] = 0;   Tcb_b[21] = 1;    
+
+    //TODO: Implement Collision Integral Data for AIR-7 (JN)
+    /*--- Collision integral data ---*/
+    //TODO: ELECTRONS HAvE NO OMEGAS....NEED TO ALTER LOOPS? or add dummy vector.....
```

Comment:
I think the general reasoning is electrons are too small to contribute meaningfully to transport properties, so their collision integrals are effectively zero. A dummy vector makes sense I think.

Reply:
👍 Ill pull more from Scalabrin and park during the viscous implementation.